examples/haskell/example.hs

```diff
@@ -0,0 +1,16 @@
+
+import OpenCog.AtomSpace.Api
+import OpenCog.AtomSpace.Types
+import Control.Monad.IO.Class
+import Data.Default
+
+main :: IO ()
+main = runOnNewAtomSpace program
+
+program :: AtomSpace ()
```

Comment:
@MarcosPividori Just to be sure I understand correctly, if createAtomSpace, addNode, etc binds to the C++ implementation, then those functions would still involve an IO monad in their types, right? Due to using FFI, even if the AtomSpace is passed in argument to addNode, etc, right?